src/config.py

```
@@ -948,6 +954,13 @@ class StreamingConfig(BaseModel):
     low_watermark_ms: int = Field(default=80)
     provider_grace_ms: int = Field(default=500)
     logging_level: str = Field(default="info")
+    # Development diagnostics. These fields must be declared so the resolved
+    # environment values survive Pydantic validation and reach every tap and
+    # full-call capture consumer. Capture remains opt-in by default.
+    diag_enable_taps: bool = Field(default=False)
+    diag_pre_secs: int = Field(default=1, ge=0, le=60)
+    diag_post_secs: int = Field(default=1, ge=0, le=60)
+    diag_out_dir: str = Field(default="/tmp/ai-engine-taps")
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/ai-engine-taps"

(S108)